go.mod

```diff
@@ -11,6 +11,8 @@ require (
 	github.com/google/uuid v1.6.0
 	github.com/onsi/ginkgo/v2 v2.23.4
 	github.com/onsi/gomega v1.37.0
+	github.com/pkoukk/tiktoken-go v0.1.7
```

Comment:
@Xunzhuo it's great to see you getting involved in the project! Why use `github.com/pkoukk/tiktoken-go` instead of `github.com/tiktoken-go/tokenizer`? The latter seems to be better maintained.